pylint/checkers/base/comparison_checker.py

```diff
@@ -58,6 +59,13 @@ class ComparisonChecker(_BasicChecker):
             "comparison-with-itself",
             "Used when something is compared against itself.",
         ),
+        "R0125": (
+            "Comparison between constants: '%s %s %s' is also a constant, remove the comparison and consider a refactor",
+            "comparison-of-constants",
+            "When two literals are compared with each other the result is a constant, "
+            "and using the constant directly is both easier to read and more performant."
+            " Initializing 'True' and 'False' this way is not required since python 2.3",
```

Comment:
```suggestion
            "When two literals are compared with each other the result is a constant. "
            "Using the constant directly is both easier to read and more performant. "
            "Initializing 'True' and 'False' this way is not required since Python 2.3.",
```